### DISCUSSION TITLE
Drainage Area raster without values

### DISCUSSION THREAD

**Discussion by sshahverdian** [Riverscapes/riverscapes-tools#448] [Channel Area] [answered]
When using the Identify tool in ArcGIS to learn a drainage area value, the pixel value comes up 0.000, for all pixels. The symbology appears to know the values of each pixel (it changes along the drainage network) but the identify tool isn't picking it up.
![drainage area - no value](https://user-images.githubusercontent.com/21205131/135509603-62555c30-d0f1-4a79-bfd1-d03a6a5b6423.png)

**Comment by philipbaileynar** [comment]
The best way to get drain areas in riverscapes context is to use the NHD flow lines and use the TotDA_sqkm attribute field. Do not rely on the rasters.

**Comment by sshahverdian** [comment]
Good to know. Thanks.

**Reply by joewheaton** [reply]
@sshahverdian what project (link) and specific layer were you looking at? While @philipbaileynar may be right, I'd like to find out what is going on and if behavior is duplicated in QRAVE.

**Reply by sshahverdian** [reply]
@joewheaton [RS context project](https://data.riverscapes.xyz/#/PUBLIC_DATA/cb56a084-6bfe-49df-b432-b5cdd4dcc7ba). A quick visit to the other side of JQL showed that Cashe and Tanner experienced the same issue for two different projects. I did not have the same problem in QRAVE - the appropriate value did show up.

**Reply by joewheaton** [reply]
> @joewheaton [RS context project](https://data.riverscapes.xyz/#/PUBLIC_DATA/cb56a084-6bfe-49df-b432-b5cdd4dcc7ba). A quick visit to the other side of JQL showed that Cashe and Tanner experienced the same issue for two different projects. I did not have the same problem in QRAVE - the appropriate value did show up.

Thanks @sshahverdian that is odd. @philipbaileynar lets (as a very low priority) poke around on this one.

**Comment by joewheaton** [comment, marked as answer]
@sshahverdian we don't know when we'll fix this. Thanks for brining to our attention. We have logged this as bug in #492
